cityofaustin/atd-moped
# Rename "Contractors" column on project list to "Workgroup/Contractors"

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/14249

I renamed the `contractors` column on the project list to `workgroup_contractors`, as well as renaming the `contractor` column on the `moped_proj_work_activity` table to be be `workgroup_contractor`. 

I also snuck in making the task orders in the project list view to be distinct, so they are not repeated in the project list table.

## Testing
**URL to test:** 


Test locally

**Steps to test:**

spin up the editor. The column previously titled "Contractors" on the project list is now titled "Workgroup/Contractors"

Go to a project and test the renaming of the contractor column by:
1. Navigating to the funding tab. Add a new work activity, and populate the workgroup/contractor field by either selecting a workgroup from the dropdown or typing in a new contractor name. Save. 
2. Confirm the activity appears on the funding tab, with expected workgroup contractor.
3. Edit an activity, changing the name of the contractor, confirm the update mutation is also working. 
4. Duplicate some task orders (different work activities, same task order) and check that the task orders only appear once in the Projects list. 


---
#### Ship list
